api/hubble/enriched_history_operations.mdx

```diff
@@ -0,0 +1,134 @@
+| Column Name | Type | Description |
+| --- | --- | --- |
+| op_id | int64 | Unique identifier for an operation.<br /><br />- Primary Key<br />- Natural Key<br />- Cluster Field<br />- Required Field<br /><br />#### Notes:<br />The operation id is the transaction id + order number |
+| op_source_account | string | The account address that originates the operation<br /><br />- Cluster Field<br />- Required Field<br /><br />#### Notes:<br />Defaults to '' |
+| op_source_account_muxed | string | If an account is multiplexed (muxed), the virtual account address that originates the operation |
+| transaction_id | int64 | A unique identifier for this transaction.<br /><br />- Primary Key<br />- Natural Key<br />- Cluster Field<br />- Required Field |
+| type | int64 | The number indicating which type of operation this operation executes<br /><br />- Required Field |
+| type_string | string | The type of operation this operation executes<br /><br />- Required Field |
```

Comment:
I think this would be better as "String representation of the operation executed"